File: Even_Fibonacci_Numbers/EvenFibonacciNumbers.cs
using System;

namespace EvenFibonacciNumbers
{
	class MainClass
	{
		public static void Main(string[] args)
		{
			if (args.Length != 3)
			{
				Console.WriteLine("This program requires 2 starting integers, and the max terms to count!\n");
				Environment.Exit(0);
			}

			int start1 = Convert.ToInt32(args[0]);
			int start2 = Convert.ToInt32(args[1]);
			int maxCount = Convert.ToInt32(args[2]);
			if (maxCount < 2)
				maxCount = 2;

			int[] fullList = new int[maxCount];
			fullList[0] = start1;
			fullList[1] = start2;

			for (int i = 2; i < maxCount; i++)
				fullList[i] = fullList[i - 1] + fullList[i-2];


			Console.Write("Result - {");
			for (int i = 0; i < fullList.Length; i++)
			{
				if (fullList[i] % 2 == 0)
				{
					Console.Write($" {fullList[i]}");
				}
			}
			Console.Write(" }");

			Console.WriteLine();
			Console.ReadKey();
		}
	}
}

File: Even_Fibonacci_Numbers/EvenFibonacciNumbers.cpp
#include <iostream>
#include <vector>

using namespace std;

int main(int argc, char* argv[]) {

	if (argc != 4) {
		cout << "This program requires 2 starting integers, and the max terms to count!" << endl;
		exit(0);
	}

	int start1 = atoi(argv[1]);
	int start2 = atoi(argv[2]);
	int maxCount = atoi(argv[3]);
	if (maxCount < 2)
		maxCount = 2;

	vector<int> fullList;
	fullList.push_back(start1);
	fullList.push_back(start2);
	fullList.resize(maxCount);

	for (int i = 2; i < maxCount; i++) {
		fullList[i] = fullList[i - 1] + fullList[i - 2];
	}
	cout << "Result - {";
	for (int fib : fullList) {
		if (fib % 2 == 0) {
			cout << " " << fib;
		}
	}
	cout << " }" << endl;

	system("pause");
	return 0;
}
